cmd/kubeadm/app/cmd/certs_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build !windows
+// +build !windows
```

Comment:
can these be without the space after // and before +  ?

Reply:
Updated. 

It seems to be VS-Code default auto-formating adding space.

Reply:
Updated. 

It seems to be VS-Code default auto-formating adding space.

Reply:
it seems the space is common
https://dave.cheney.net/2013/10/12/how-to-use-conditional-compilation-with-the-go-build-tool
https://www.digitalocean.com/community/tutorials/customizing-go-binaries-with-build-tags

we should have it. sorry about that.